tasks/test.py

```diff
@@ -432,3 +432,23 @@ def is_unwanted(job, version):
                 for need in needed:
                     if is_unwanted(data[need], version):
                         print("{} needs on {} but it won't be built for A{}".format(k, need, version))
+
+@task
+def install_shellcheck(ctx, version="0.7.0", destination="/usr/local/bin"):
+    """
+    Installs the requested version of shellcheck in the specified folder (by default /usr/local/bin).
+    Required to run the shellcheck pre-commit hook.
+    """
+
+    if sys.platform == 'win32':
+        print("shellcheck is not supported on Windows")
+        return 1
```

Comment:
The `invoke` page recommends using [a custom exception](http://docs.pyinvoke.org/en/stable/api/exceptions.html#invoke.exceptions.Exit) to exit on failure. This is used in other tasks in this file already so it should be as simple as:
```suggestion
        raise Exit(code=1)
```